tools/distpartitioning/dataset_utils.py

```diff
@@ -33,46 +33,63 @@ def get_dataset(input_dir, graph_name, rank, num_node_weights):
     """
     #node features dictionary
     node_features = {}
+    node_feature_tids = {}
 
-    #iterate over the sub-dirs and extract the nodetypes
-    #in each nodetype folder read all the features assigned to 
-    #current rank
-    siblings = os.listdir(input_dir)
-    for s in siblings:
-        if s.startswith("nodes-"):
-            tokens = s.split("-")
-            ntype = tokens[1]
-            num_feats = tokens[2]
-            for idx in range(int(num_feats)):
-                feat_file = s +'/node-feat-'+'{:02d}'.format(idx) +'/'+ str(rank)+'.npy'
-                if (os.path.exists(input_dir+'/'+feat_file)):
-                    features = np.load(input_dir+'/'+feat_file)
-                    node_features[ntype+'/feat'] = torch.tensor(features)
+    #iterate over the "node_data" dictionary in the schema_map
+    #read the node features if exists
+    #also keep track of the type_nids for which the node_features are read.
+    dataset_features = schema_map["node_data"]
```

Comment:
Same problem.

Reply:
Changed the code appropriately so that the function invocation and function signature match w.r.t to the argument list.